### PR TITLE
Close #128: Add (and use) definitions for SDF models and documents

### LINKED ISSUE #128: Add terminology entries for SDF model and SDF document

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -263,15 +265,29 @@ Declaration:
...
 
+SDF Document:
+: Container for SDF Definitions, together with data
+  about the SDF Document itself (information block).
+  Represented as a JSON text representing a single JSON map, which is
+  built from nested maps.
+
+SDF Model:
+: Definitions and declarations that describe the digital interaction
+  opportunities offered by one or more classes of Things, represented
```
Comment (JKRhb) [inline sdf.md:276]: Is "capabilities" maybe a better fit than "opportunities" here?

**Review by JKRhb — APPROVED** [review]
Changes look good to me, see only one minor comment below :)

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md:276]: I like "opportunities", as that is in line with the "affordance" style of thinking, but "capabilities" would work as well, even if this is loaded with a different bag of connotations (why I don't prefer it).

File: sdf.md [hunk 2]
```diff
@@ -173,7 +173,9 @@ document that a specific extension is in effect; see
 {{information-block}} for details of the `feature` quality that can be
 used for such indications.
 Base SDF does not define a "version" concept for the SDF specification
-itself (as opposed to SDF documents); however there may be occasional
+itself (as opposed to version indications for the specifications
```
Comment (akeranen) [inline sdf.md:176]: Is “specifications in SDF documents” terminology we use elsewhere? (On mobile now, difficult to check)

**Review by akeranen — APPROVED** [review]
changes LGTM except perhaps for the one question above

File: sdf.md (hunk 1 above)
Comment (JKRhb) [reply sdf.md:276]: I see, that makes sense, thank you :)

File: sdf.md (hunk 2 above)
Comment (cabo) [reply sdf.md]: See below ("Improve...").

**Pull request merged**